docs/topics/feed-exports.rst

```diff
@@ -319,3 +320,16 @@ A dict containing the built-in feed exporters supported by Scrapy.
 .. _URI: http://en.wikipedia.org/wiki/Uniform_Resource_Identifier
 .. _Amazon S3: http://aws.amazon.com/s3/
 .. _boto: http://code.google.com/p/boto/
+
+FEED_BATCH_SIZE
+---------------
+
+Default: ``None``
+
+Allows to control the number of item exported in one file.
```

Comment:
I think we can be more explicit and explain what happens if FEED_BATCH_SIZE is None (a single file is created) and what happens if it is a number (multiple files are created, each containing FEED_BATCH_SIZE items maximum).